src/App.css

```diff
@@ -39,3 +39,10 @@ tfoot tr:first-child {
 .baseline {
   stroke-dasharray: 4, 4;
 }
+tr:nth-child(even){background-color: #a1c5e5;}
```

Comment:
Nice, I like the idea of styling the table rows but I think something like every 5 rows would be more valuable than every other.